### DISCUSSION TITLE
Indication that a search is running when you search something from PnP search box

### DISCUSSION THREAD

**Discussion by supreet-bajaj** [microsoft-search/pnp-modern-search#2520] [Ideas]
1. To introduce some visual indication that the search is running.
Examples are: Hourglass (most common icon display during page loading), spinning circle (as in win10 startup)

2. Search result area to be cleared (i.e. showing blank) until the search is complete.
I know there is a shimmer on the first load but we would like to show Loading icon after adding filters from the Search Filters WebPart. The shimmer is not intuitive enough for our end users.

Can we do something like that in PnP search results?

**Comment by wobba** [comment]
Duplicate of #2499
